### ISSUE TITLE
Client CSR is not deleted CentOS 7

### ISSUE THREAD

**Issue by terryjrjoyce** [kyl191/ansible-role-openvpn#184]
When revoking the clients the CA cert is not removed from the server.  

**Current Code**
```
- name: Revoke certificates
  command: sh revoke.sh {{ item }}.crt
  changed_when: true
  args:
    chdir: "{{ openvpn_key_dir }}"
  with_items:
    - '{{ openvpn_revoke_these_certs }}'
    - '{{ openvpn_cert_sync_revoke | default([]) }}'
```
**New Code**
```
- name: Revoke client crt
  file:
    path: "{{ openvpn_key_dir }}/{{ item }}.crt"
    state: absent
    force: true
  with_items:
    - '{{ openvpn_revoke_these_certs }}'
    - '{{ openvpn_cert_sync_revoke | default([]) }}'
```

**Comment by jbeyerstedt** [comment]
I think this would be fixed by #186.

**Comment by Nmishin** [comment]
Merged to the forked role: https://github.com/aovpn/ansible-role-openvpn/pull/37
